apps/blockchain/scripts/sync_with_infura.ex

```diff
@@ -6,7 +6,8 @@ defmodule SyncWithInfura do
 
   def setup() do
     db = MerklePatriciaTree.DB.RocksDB.init(db_name())
-    chain = Blockchain.Chain.load_chain(:foundation)
+    config = EVM.Configuration.EIP158.new()
```

Comment:
Maybe I should remove these changes...

Reply:
Yeah, I think we should default the `sync_with_infura` to mainnet for now